config/routes.rb

```diff
@@ -1,3 +1,11 @@
 Rails.application.routes.draw do
   # For details on the DSL available within this file, see http://guides.rubyonrails.org/routing.html
+
+  resources :merchants do
+    scope module: :merchant do
+      resources :dashboard, only: [:index]
```

Comment:
This feels like it should be an action in the merchant controller instead of a whole new controller, being as the dashboard is a single view for the merchant.